storage/node.go

```diff
@@ -37,7 +37,9 @@ type Node struct {
 	NodeRevision int64
 }
 
-// NodeID uniquely identifies a Node within a MerkleTree.
+// NodeID is an immutable identifier of a Merkle tree Node. A default
```

Comment:
Technically, it's not immutable (yet) as the struct fields are exported and can be modified.

Reply:
Made this statement a bit weaker.